recipes-extended/man-pages/files/onie-bisdn-upgrade.1

```diff
@@ -1,30 +1,30 @@
 .\" Manpage for onie-bisdn-upgrade.
 .\" Contact pierluigi.greto@bisdn.de to correct errors or typos.
```

Comment:
Maybe we should update the email address here as well. Or just point to this repo.